internal/test/builder/builders.go

```diff
@@ -395,18 +395,19 @@ func (m *MachineDeploymentClassBuilder) Build() *clusterv1.MachineDeploymentClas
 }
 
 // InfrastructureMachineTemplateBuilder holds the variables and objects needed to build an InfrastructureMachineTemplate.
-// +kubebuilder:object:generate=false
```

Comment:
Removing the tag allows us to see the error we're trying to solve - to see the issue remove a tag and then run 

```
make generate-go-deepcopy
```